docs/overview.rst

```diff
@@ -0,0 +1,45 @@
+Overview
+========
+
+
+Start your own Vyper project
+----------------------------
+
+Twig lets you write tests using ``web3.py`` and ``pytest-ethereum`` against your ``Vyper`` smart contracts.
+
+
+Twig setup
+~~~~~~~~~~
+
+- clone twig repo
+- ``python -m venv venv`` (note: requires python3.6)
+- ``. venv/bin/activate``
+- ``pip install -e .``
+
+
+Project setup
+~~~~~~~~~~~~~
+
+.. NOTE:: ``twig/contracts/name_registry.vy`` and ``tests/contracts/test_name_registry.py`` have been included as examples on how to get started, but are safe to delete.
+
+- create your vyper contract file in ``twig/contracts/``
```

Comment:
curious of your choice to namespace the contracts under a `twig` directory. This seems odd to me as I haven't seen this pattern in other frameworks (requiring a framework specific top level namespace).

Reply:
there's other code in `root/twig/` (i.e. compiler / utils / backends) but I moved `root/twig/contracts/` to the project root level `root/contracts/`